src/tests/beforeSetupTests.tsx

```diff
@@ -23,3 +24,12 @@ config({ path: path.resolve(process.cwd(), '.env.example') })
 Object.assign(globalThis, { TextDecoder, TextEncoder })
 
 setCurrentLocale('en', mergeTranslations(flatten(dappsEn), flatten(locales.en)))
+
+if (!globalThis.fetch) {
```

Comment:
Is this necessary, can't the global fetch be mocked without the need of installing node-fetch?